src/main/resources/org/zaproxy/zap/extension/neonmarker/resources/Messages.properties

```diff
@@ -1,19 +1,19 @@
 neonmarker.name = Neonmarker
-
-neonmarker.panel.title = Neonmarker
-neonmarker.panel.button.clear = Remove All Rules
 neonmarker.panel.button.add = Add a Rule
-neonmarker.panel.mapping.remove = Remove Rule
-neonmarker.panel.mapping.move = Change Precedence
-neonmarker.panel.mapping.notags = No Tags Available
+neonmarker.panel.button.clear = Remove All Rules
 neonmarker.panel.color.chooser.title = Choose Custom Color
 neonmarker.panel.color.menu.custom.label = Custom
-neonmarker.panel.toolbar.toggle.button.label.enabled = Enabled
-neonmarker.panel.toolbar.toggle.button.tooltip.enabled = Click to disable coloring rules.
+neonmarker.panel.mapping.move = Change Precedence
+neonmarker.panel.mapping.notags = No Tags Available
+neonmarker.panel.mapping.remove = Remove Rule
+
```

Comment:
This empty line could be removed if you want, but otherwise looks good

Reply:
Thanks